src/components/MediaLibrary/MediaLibrary.js

```diff
@@ -316,7 +374,12 @@ class MediaLibrary extends React.Component {
                   <div className="nc-mediaLibrary-cardImage-container">
                     {
                       file.isViewableImage
-                        ? <img src={file.url} className="nc-mediaLibrary-cardImage"/>
+                        ? <CardImage
+                          isVisible={true}
```

Comment:
Doesn't look like we're using this.

Reply:
Good catch - this is leftover from when I was only displaying visible images in the media library (c.f. https://github.com/benaiah/netlify-cms/tree/experimental-media-library-scrolling).

Reply:
Yeah figured as much.